en/docs/guides/adaptive-auth/work-with-acr-amr.md

```diff
@@ -0,0 +1,167 @@
+     ```
+    	        
+6. Copy the `ID_TOKEN` and decode with Base64. The decoded `ID_TOKEN` will look similar to the following. 
+
+	``` tab="Format"
+	{
+	  "at_hash": "<<Access token hash value>>",
+	  "aud": "<<Audience that the ID token is intended for>>",
+	  "c_hash": "<<Code hash value>>",
+	  "sub": "<<Subject>>",
+	  "acr": "<<ACR>>", 
+	  "nbf": <<Epoch time before which the JWT must not be accepted for processing>>,
+	  "azp": "<<Authorized party>>",
+	  "amr": [<<Authentications Method Reference>>],
+	  "iss": "<<The token endpoint that issued the token>>",
+	  "exp": <<Epoch time of the token expiration date/time>>,
+	  "iat": <<Epoch time of the token issuance date/time>>
+	}
+	```
+
+	``` tab="Example"
+	{
+      "at_hash": "6OXwfxJaTWYC56RccEhSJg",
+      "aud": "EUVvhKM28RkwTQL9A52kqXnfCj8a",
+      "acr": "LOA2",
+      "c_hash": "lDj9nihZGSUmgNmz_lxxXA",
+      "sub": "admin",
+      "nbf": 1548396413,
+      "azp": "EUVvhKM28RkwTQL9A52kqXnfCj8a",
+      "amr": [
+              "DemoFaceIdAuthenticator",
+              "BasicAuthenticator",
+              "DemoFingerprintAuthenticator"
+      ],
+      "iss": "https://localhost:9443/oauth2/token",
+      "exp": 1548400013,
+      "iat": 1548396413
+    }
+	```
+	
+	Note that the AMR values in this example are `DemoFaceIdAuthenticator`, `BasicAuthenticator`, and `DemoFingerprintAuthenticator`.
+	These are the authenticators that are used in the authentication process.
+
+              
+!!! note
+    Following is a sample SAML fragment that you can add to the SAML SSO request while initiating the SAML SSO flow. 
+
+    ```
+    <samlp:RequestedAuthnContext Comparison="exact">
+    <saml:AuthnContextClassRef>urn:federation:authentication:windows</saml:AuthnContextClassRef>
+    <saml:AuthnContextClassRef>pwd</saml:AuthnContextClassRef>
+    <saml:AuthnContextClassRef>LOA2</saml:AuthnContextClassRef>
+    </samlp:RequestedAuthnContext>
+    ```
```

Comment:
can this be an optional step instead of a note?

Reply:
This is not an optional step. This is a note on how to send a request with SAML.